lapce-proxy/src/lib.rs

```diff
@@ -23,25 +23,31 @@ use lapce_rpc::{
     stdio::stdio_transport,
     RpcMessage,
 };
+use unzip3::Unzip3;
 
 #[derive(Parser)]
 #[clap(name = "Lapce")]
 #[clap(version=*meta::VERSION)]
 struct Cli {
     #[clap(short, long, action)]
     proxy: bool,
-    paths: Vec<PathBuf>,
+    #[derive(Unzip3)]
+    references: Vec<PathBuf>,
 }
 
 pub fn mainloop() {
     let cli = Cli::parse();
+    let (paths, _lines, _columns) = cli.references
+        .iter()
+        .map(|r| r.split(':')) // split reference into Vec<path, line, column>
+        .map(|v| match v.as_slice() {
+            [path, line, column] => (path, line, column),
+            _ => panic!("Arguments must have the format: \"file:line:column\", or \"file:line\", or \"file\" ");
+        })
+        .unzip3();
```

Comment:
Is this required?

Reply:
I think I have a better approach, see latest commit